File: NoBSMealPrep/ClientApp/src/interfaces/GroceryList.ts
import { FavoriteRecipe } from "./FavoriteRecipe";

export interface GroceryList {
    id: number;
    food: string;
    quantity: number;
    foodCategory: string;
    measure: string;
    parentRecipe: number;
    parentRecipeNavigation: FavoriteRecipe;
}

File: NoBSMealPrep/Models/GroceryList.cs
using System;
using System.Collections.Generic;

namespace NoBSMealPrep.Models;

public partial class GroceryList
{
    public int Id { get; set; }

    public string? Food { get; set; }

    public double? Quantity { get; set; }

    public string? Measure { get; set; }

    public int? ParentRecipe { get; set; }

    public virtual FavoriteRecipe? ParentRecipeNavigation { get; set; }
}
